packages/common/index.ts

```diff
@@ -276,13 +276,13 @@ export type GetQuestionResponse = Question;
 
 export class CreateQuestionParams {
   @IsString()
-  text: string;
+  text!: string;
 
   @IsEnum(QuestionType)
-  questionType: QuestionType;
+  questionType!: QuestionType;
 
   @IsInt()
-  queueId: number;
+  queueId!: number;
```

Comment:
Just curious, what is the ! for?

Reply:
tells typescript to not freak out that the value might be null able, cause the value is never nullable